src/main/java/mate/academy/spring/dto/MovieRequestDto.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.spring.dto;
+
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class MovieRequestDto {
```

Comment:
```suggestion
public class MovieRequestDto {
```
dto classes should not be annotated with component annotation